app/controllers/welcome_controller.rb

```diff
@@ -15,6 +15,13 @@ def popular
     renders(@events)
   end
 
+  def about
+    @attendees = Attendee.count
+    @tickets = UserTicket.count
```

Comment:
`@attendees` and `@tickets` as variable names does not rightly reflect what these variables are, which are counts. Kindly check variable namings.